home.html

```diff
@@ -0,0 +1,171 @@
+          <div class="home-page-main-program-border"></div>
+        </div>
+        <div class="home-page-main-program-container">
+          <div class="home-page-main-program-1">
+            <img src="image/program_icon_01.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-1-oratory">Lecture</h2>
+            <p class="home-page-main-program-paragraph"> Listen to the speakers from various countries about the messages of sharing and opening.</p>
+          </div>
+          <div class="home-page-main-program-2">
+            <img src="image/program_icon_02.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-2-oratory">Exhibition</h2>
+            <p class="home-page-main-program-paragraph"> Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+          </div>
+          <div class="home-page-main-program-3">
+            <img src="image/program_icon_03.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-3-oratory">Forum</h2>
+            <p class="home-page-main-program-paragraph"> Have the time to share your thoughts and opinions with experts for each topic.</p>
+          </div>
+          <div class="home-page-main-program-4">
+            <img src="image/program_icon_04.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-4-oratory">Workshop</h2>
+            <p class="home-page-main-program-paragraph"> Try creating your own work using open source license rather than just watching at it.</p>
+          </div>
+          <div class="home-page-main-program-5">
+            <img src="image/program_icon_05.png" alt="" class="home-page-main-program-img">
+            <h2 class="home-page-main-program-5-oratory">Ignite</h2>
+            <p class="home-page-main-program-paragraph"> get opportunities to network with CC affiliates around the world, also after the summit.</p>
+          </div>
+        </div>
+
+          <!--    button to submit    -->
+
+        <button class="home-page-info-joinNowBtn">GET YOUR PASS</button>
+
+      </div>
+    </section>
+      
+      <!--    speaker me page   -->
+    
+    <section class="speaker-page">
+      <h2 class="speaker-page-h1">Featured Speaker</h2>
+      <div class="speaker-page-h1-div"></div>
+      <div class="speaker-page-div">
+      </div>
+    </section>
+    <section class="sponsor-page">
+      <div class="sponsor-page-past-sponsors">
+        <div><h2>Our Partners</h2></div>
+        <div class="sponsor-page-past-div"></div>
+        <div class="sponsor-img-1"><img src="image/airbnb 1.png" alt=""><img src="image/mozilla 1.png" alt=""><img src="image/naver 1.png" alt=""></div>
+        <div class="sponsor-img-2"><img src="image/googlekorea 1.png" alt=""><img src="image/daumkakaologo 1.png" alt=""></div>
+      </div>
+    </section>
```

Comment:
- [ ] kindly consider making sure that this section looks exactly like the one provided in the design to make your work an exact duplicate of the given design
- your design 👇 
<img width="750" alt="Screen Shot 2023-05-25 at 07 29 01" src="https://github.com/Zven94/conference-page/assets/60146030/5e974c9d-c32e-46cf-a202-397250917c37">


1. consider displaying all 5 partners on the same line as in the given design 
2. consider adding the red line under the our partners title